coldfront/core/allocation/views.py

```diff
@@ -1234,54 +1230,6 @@ def test_func(self):
         messages.error(self.request, 'You do not have permission to manage invoices.')
         return False
 
-    def get_context_data(self, **kwargs):
```

Comment:
I believe we want to keep this function has it contains all the business logic? and delete the other one?